build.ts

```diff
@@ -299,6 +309,18 @@ function processBanc(filePath: string, mapType: string, mapName: string) {
           }
         }
       }
+      // If DropTable and DropActor do not exist and an blank drop exists
+      //   set the DropTable name to as 'Default'
+      if (!dyn.Drop__DropTable && !dyn.Drop__DropActor) {
+        if (actor.Gyaml in DROP_TABLES) {
+          for (const table of DROP_TABLES[actor.Gyaml]) {
+            if (table.DropTableName == "") {
+              drops.push(2);
```

Comment:
what is the 2? and can we get rid of it?

Reply:
I added `const`s for `DropTable = 2` and `DropActor = 1`.  We can change them to strings if desired.  Those values are also on the front-end.

Reply:
I think this makes parsing rather awkward - can we turn drops into an array of `{type: DROP_TABLE / DROP_ACTOR, value: xxx}` objects?

Reply:
also we probably want to give those constants a prefix, e.g. DROP_TYPE_DROP_TABLE and DROP_TYPE_ACTOR

Reply:
Shape of drops has changed to 
`{ type: string, value: string[] }`
- type is either "Actor" or "Table"
- value is [ table_name ] when type = Table
- value is [ actor, actor ... ] when type = Actor